packages/gitbook/src/components/DocumentView/Blocks.tsx

```diff
@@ -65,7 +65,7 @@ export function UnwrappedBlocks<TBlock extends DocumentBlock>(props: UnwrappedBl
 
         return (
             <Block
-                key={node.key}
+                key={node.key || `${node.type}-${index}`}
```

Comment:
What is the case where we don't have a `key`? it indicates that we do `serialize()` without `preserveKey` in GBX, not a big issue but we need to make sure we don't depend on it anywhere else

Reply:
The type is `key?` so this is needed, but yeah we should have keys here.